server/helpers/loadFrontendConfig.js

```diff
@@ -4,13 +4,13 @@
 
 function getBlockchainExplorerUrlForFrontend() {
   return process.env.CARDANOLITE_ENABLE_MOCK_SERVER === 'true'
-    ? `http://localhost:${process.env.PORT}`
+    ? process.env.CARDANOLITE_SERVER_URL
```

Comment:
I would expecting this to have a name `CARDANOLITE_BLOCKCHAIN_EXPLORER_MOCK_URL` because when you will look into code where else it is used, you will see that it`s use is not just for mocking..